apps/web/playwright/booking-pages.e2e.ts

```diff
@@ -80,12 +80,6 @@ testBothFutureAndLegacyRoutes.describe("free user", () => {
     await expect(page.locator("[data-testid=success-page]")).toBeVisible();
     const { title: eventTitle } = await user.getFirstEventAsOwner();
 
-    await expectEmailsToHaveSubject({
-      emails,
-      organizer: user,
-      booker: bookerObj,
-      eventTitle,
-    });
```

Comment:
this works fine locally but fails on CI.

Reply:
added a new unit test "cannot book same slot multiple times" in fresh-booking to make sure which would check the email has been delivered to both booker and organizer